src/main/java/com/kk/strong/repository/ExerciseRepository.java

```diff
@@ -1,10 +1,14 @@
 package com.kk.strong.repository;
 
 import com.kk.strong.model.Exercise;
+import com.kk.strong.model.ExerciseType;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface ExerciseRepository extends JpaRepository<Exercise, Long> {
 
+    List<Exercise> findAllByWorkoutSession_User_IdAndExerciseTypeOrderByWorkoutSession_Timestamp(Long userId, ExerciseType exerciseType);
```

Comment:
I don't like how it looks :P Is declaring such methods preffered over handling the filtering inside a service method?